docs/src/python/user-guide/sql/intro.py

```diff
@@ -0,0 +1,18 @@
+# --8<-- [start:setup]
+import polars as pl
+
+# --8<-- [end:setup]
+
+# --8<-- [start:context]
+sql = pl.SQLContext()
+# --8<-- [end:context]
+
+
+# --8<-- [start:register]
+pokemon = pl.read_csv(
```

Comment:
`read_csv().lazy()` is an anti-pattern. We only need this because the data needs to fetched from the internet here.

Maybe we should add a note that for local files, one always must use  `pl.scan_csv` to ensure optimizations can be pushed into the readers.